1.10/installing/custom/configuration/configuration-parameters.md

```diff
@@ -501,3 +501,31 @@ Indicates whether to enable the DC/OS proxy.
 For more information, see the [examples](/docs/1.10/installing/custom/configuration/examples/#http-proxy).
 
 **Important:** You should also configure an HTTP proxy for [Docker](https://docs.docker.com/engine/admin/systemd/#/http-proxy).
+
+<a id="zk-superuser"></a> 
+### zk_super_credentials (Enterprise DC/OS Only)
```

Comment:
This is now "DC/OS Enterprise" and not "Enterprise DC/OS" I believe.

Reply:
@adamtheturtle Yes, but we are not doing this wholesale renaming until we deploy the new merged DC/OS Enterprise and DC/OS open source doc sites.